client/my-sites/checkout/composite-checkout/components/wp-checkout-order-review.js

```diff
@@ -121,6 +125,53 @@ function CouponFieldArea( {
 	);
 }
 
+function MonthlyPricingTestWrapper( { children, items } ) {
+	const translate = useTranslate();
+
+	return (
+		<Experiment name="monthly_pricing_test_phase_1">
+			<DefaultVariation name="control">
+				{ React.Children.map( children, ( child ) =>
+					React.cloneElement( child, { isMonthlyPricingTest: false } )
+				) }
+			</DefaultVariation>
+			<Variation name="treatment">
+				{ React.Children.map( children, ( child ) =>
+					React.cloneElement( child, {
+						isMonthlyPricingTest: true,
+						items: items.map( ( item ) => overrideItemSublabel( item, translate ) ),
+					} )
+				) }
+			</Variation>
+		</Experiment>
+	);
+}
+
+/*
+ * WARNING:
+ * Please update item's sublabel in the `translateReponseCartProductToWPCOMCartItem` function
+ * when the treatment is rolled out as a winner.
+ */
+function overrideItemSublabel( item, translate ) {
+	if ( item?.type !== 'plan' || ! isWpComPlan( item?.wpcom_meta?.product_slug ) ) {
+		return item;
+	}
+
+	const billingMonths = getBillingMonthsForPlan( item?.wpcom_meta?.product_slug );
+
+	let sublabel = translate( 'Monthly subscription' );
+	if ( billingMonths === 12 ) {
+		sublabel = translate( 'One year subscription' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'for a one year subscription' )` *ES Score: 8*